ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -63,6 +63,7 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     ...    workflow_name=${workflow_name}
     Delete Pipeline Run    ${PIPELINE_TEST_RUN_BASENAME}    ${PIPELINE_TEST_NAME}
     Delete Pipeline    ${PIPELINE_TEST_NAME}
+    Delete Pipeline Server    ${PRJ_TITLE}
```

Comment:
could check that the pipeline pods have been deleted after the server deletion?

Reply:
Done